src/murmur/Server.cpp

```diff
@@ -1110,6 +1110,14 @@ void Server::processMsg(ServerUser *u, const char *data, int len) {
 
 	len = pds.size() + 1;
 
+	if (poslen > static_cast<unsigned int>(len)) {
```

Comment:
I assume the cast is needed due to the compiler complaining about unsigned & signed comparison, right?

Hopefully once we switch to POCO we don't have to deal with such cases anymore...

Reply:
Yes exactly. How would POCO solve this issue though. signed and unsigned integers are different things. POCO doesn't change that :thinking:

Reply:
I'm pretty sure POCO uses `size_t`.

Reply:
Ah okay - yeah that'd probably solve the issue here :)